.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
As the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#requirements) the should be three files you have two of them great 👍 but you missed adding `test.md` file and include it here in the `.gitignore` file as you did for the `node_modules`. Kindly, add this file to you project and make sure it's ignored.

<img width="925" alt="image" src="https://user-images.githubusercontent.com/34578631/170242069-ef6df243-e9a0-42e6-b6c1-9d0ef950e698.png">